node/test/mocktests.ts

```diff
@@ -183,5 +183,56 @@ describe('Mock Tests', function () {
         
         assert(tool, "tool should not be null");
         assert(rc == 0, "rc is 0");
-    })                
+    })
+
+    it('Mock toolRunner returns success code for execTool', async () => {
+        var a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
+            "execTool": {
+                "/usr/local/bin/atool": (args: string[]) => {
+		    assert.equal(args.indexOf("--arg"), 0, "--arg should be the first element");
+		    assert.equal(args.indexOf("foo"), 1, "foo should be the second element");
```

Comment:
I see. If you want to plug in your own function, another way is with registerMockExport. Originally the answers came from a file, so the answers interfaces is currently simple types.